2023-11-26 00:01:40,066 INFO    MainThread:50660 [wandb_setup.py:_flush():76] Current SDK version is 0.15.8
2023-11-26 00:01:40,067 INFO    MainThread:50660 [wandb_setup.py:_flush():76] Configure stats pid to 50660
2023-11-26 00:01:40,067 INFO    MainThread:50660 [wandb_setup.py:_flush():76] Loading settings from /Users/marconanka/.config/wandb/settings
2023-11-26 00:01:40,067 INFO    MainThread:50660 [wandb_setup.py:_flush():76] Loading settings from /Users/marconanka/BioMedia/code/contrastive-learning/wandb/settings
2023-11-26 00:01:40,067 WARNING MainThread:50660 [wandb_setup.py:_flush():76] Unknown environment variable: WANDB_CACHE_DIR
2023-11-26 00:01:40,067 WARNING MainThread:50660 [wandb_setup.py:_flush():76] Unknown environment variable: WANDB_CONFIG_DIR
2023-11-26 00:01:40,067 WARNING MainThread:50660 [wandb_setup.py:_flush():76] Unknown environment variable: WANDB_TEMP
2023-11-26 00:01:40,067 INFO    MainThread:50660 [wandb_setup.py:_flush():76] Loading settings from environment variables: {'root_dir': '$HOME/wandb_tmp'}
2023-11-26 00:01:40,067 INFO    MainThread:50660 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program_relpath': 'contrastive.py', 'program': '/Users/marconanka/BioMedia/code/contrastive-learning/contrastive.py'}
2023-11-26 00:01:40,067 INFO    MainThread:50660 [wandb_setup.py:_flush():76] Applying login settings: {'api_key': '***REDACTED***'}
2023-11-26 00:01:40,067 INFO    MainThread:50660 [wandb_setup.py:_flush():76] Applying login settings: {'api_key': '***REDACTED***'}
2023-11-26 00:01:40,067 INFO    MainThread:50660 [wandb_init.py:_log_setup():507] Logging user logs to $HOME/wandb_tmp/wandb/run-20231126_000140-h6lb5wzw/logs/debug.log
2023-11-26 00:01:40,067 INFO    MainThread:50660 [wandb_init.py:_log_setup():508] Logging internal logs to $HOME/wandb_tmp/wandb/run-20231126_000140-h6lb5wzw/logs/debug-internal.log
2023-11-26 00:01:40,068 INFO    MainThread:50660 [wandb_init.py:init():547] calling init triggers
2023-11-26 00:01:40,068 INFO    MainThread:50660 [wandb_init.py:init():554] wandb.init called with sweep_config: {}
config: {'num_epochs': 500, 'batch_size': 14, 'learning_rate': 0.002, 'patch_size': [0, 0, 8], 'patience': 100, 'training type': 'CONTRASTIVE', 'contrastive type': 'domainda', 'image_type': 'CT', 'filter': None, 'model_name': 'domain_da_adj0_norm_ct', 'folder_path': '../data/entire_dataset_split_ts_tr/ct_pre/'}
2023-11-26 00:01:40,068 INFO    MainThread:50660 [wandb_init.py:init():596] starting backend
2023-11-26 00:01:40,068 INFO    MainThread:50660 [wandb_init.py:init():600] setting up manager
2023-11-26 00:01:40,075 INFO    MainThread:50660 [backend.py:_multiprocessing_setup():106] multiprocessing start_methods=spawn,fork,forkserver, using: spawn
2023-11-26 00:01:40,080 INFO    MainThread:50660 [wandb_init.py:init():606] backend started and connected
2023-11-26 00:01:40,098 INFO    MainThread:50660 [wandb_init.py:init():697] updated telemetry
2023-11-26 00:01:40,179 INFO    MainThread:50660 [wandb_init.py:init():730] communicating run to backend with 60.0 second timeout
2023-11-26 00:01:40,776 INFO    MainThread:50660 [wandb_run.py:_on_init():2180] communicating current version
2023-11-26 00:01:40,883 INFO    MainThread:50660 [wandb_run.py:_on_init():2189] got version response upgrade_message: "wandb version 0.16.0 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2023-11-26 00:01:40,883 INFO    MainThread:50660 [wandb_init.py:init():781] starting run threads in backend
2023-11-26 00:01:40,993 INFO    MainThread:50660 [wandb_run.py:_console_start():2159] atexit reg
2023-11-26 00:01:40,994 INFO    MainThread:50660 [wandb_run.py:_redirect():2014] redirect: wrap_raw
2023-11-26 00:01:40,994 INFO    MainThread:50660 [wandb_run.py:_redirect():2079] Wrapping output streams.
2023-11-26 00:01:40,994 INFO    MainThread:50660 [wandb_run.py:_redirect():2104] Redirects installed.
2023-11-26 00:01:40,996 INFO    MainThread:50660 [wandb_init.py:init():822] run started, returning control to user process
2023-11-26 00:01:48,851 WARNING MsgRouterThr:50660 [router.py:message_loop():77] message_loop has been closed
